test/e2e/metamask-ui.spec.js

```diff
@@ -292,12 +292,13 @@ describe('MetaMask', function () {
     it('transitions to the confirm screen', async function () {
       // Continue to next screen
       await driver.delay(largeDelayMs);
```

Comment:
If this fix works, we can remove the `await driver.delay(largeDelayMs);`, and in the below `submits the transaction` test as well